studio/schemas/documents/aboutSection.js

```diff
@@ -0,0 +1,25 @@
+export default {
+    name: 'aboutSection',
+    title: 'About Section',
+    type: 'document',
+    fields: [
+        {
+            name: 'leftAboutText',
+            title: 'Left About Text',
+            type: 'array',
+            of: [{ type: 'block' }],
+        },
+        {
+            name: 'aboutIconText',
+            title: 'About Icon Text',
+            type: 'array',
+            of: [{ type: 'reference', to: { type: 'aboutIcon' } }],
+        },
```

Comment:
this should be included in the body text for each of these columns, not it's own separate field.